File: 2.0/php/contacts/removeContactListItem.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->removeContactListItem();
        $request->getOperationConfig()->setPathParameters(array("id" => 11646003,
                                                                "contactId" => 22277003));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

File: 2.0/java/contacts/removeContactListItem.java
import com.callfire.api.client.CallfireClient;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        client.contactListsApi().removeListItem(11646003L, 22277003L);
    }
}

